config-thunderjet.yml

```diff
@@ -0,0 +1,70 @@
+okapi-url: http://localhost:9130
+registry-url: http://folio-registry.dev.folio.org
+install-json-path: /home/x/projects/folio/busybee/platform-complete/install.json  # A file path to install.json can also be used
+env-vars: # any key-pairs added here will be added to the OKAPI env service
+  DB_PASSWORD: folio_admin
+  DB_USERNAME: folio_admin
+  DB_DATABASE: okapi_modules
+  DB_HOST: host.docker.internal
+  DB_PORT: 5432
+  OKAPI_URL: http://host.docker.internal:9130
+  KAFKA_HOST: host.docker.internal
+  KAFKA_PORT: 9092
+  ELASTICSEARCH_URL: http://host.docker.internal:9200
+  SYSTEM_USER_PASSWORD: system_user_password
+  AWS_URL: http://host.docker.internal:9000
+  AWS_REGION: us-east-1
+  AWS_BUCKET: example-bucket
+  AWS_ACCESS_KEY_ID: minioadmin
+  AWS_SECRET_ACCESS_KEY: minioadmin
+  S3_IS_AWS: false
+  S3_URL: http://host.docker.internal:9000                                                                                                                                                                       
+  S3_REGION: us-east-1                                                                                                                                                                                                                                                                                                                                   
+  S3_ACCESS_KEY_ID: minioadmin                                                                                                                                                                  
+  S3_SECRET_ACCESS_KEY: minioadmin
+  S3_BUCKET: example-bucket
+  SPLIT_FILES_ENABLED: true
+be-modules: # list the FOLIO backend modules you would like to spin up
+  ## CORE: ensure that core FOLIO modules are first on this list
+  - mod-permissions
+  - mod-configuration
+  - mod-login
+  - mod-users
+  - mod-users-bl
+  - mod-authtoken
+  ## END CORE
+  - mod-source-record-manager
+  - mod-source-record-storage
```

Comment:
mod-source-record-storage and mod-source-record-manager need for Data Import functionality, so we can remove from Thunderjet config to speedup deployment time.
mod-pubsub also not needed

Reply:
edge-orders, mod-ebsconet, mod-gobi better to have to be able to run all our karate tests

Reply:
we also own functionality for Edifact exports and have karate tests for it, so can include these modules in config:
mod-data-export-spring, mod-data-export-worker but also we need to figure out how to setup MinIO, so to not block this PR lets do this in separate PR